src/abstractdataframe/selection.jl

```diff
@@ -198,6 +205,10 @@ using `_`; if more than three columns are passed then the name consists of the
 first two names and `etc` suffix then, e.g. `[:a,:b,:c,:d] => fun` produces
 the new column name `:a_b_etc_fun`.
 
+As a special rule passing `nrow` as an argument creates a column named `:nrow`
```

Comment:
Maybe that's more explicit?
```suggestion
As a special rule passing `nrow` without specifying `old_column` creates a column named `:nrow`
```